test/e2e/gameserver_test.go

```diff
@@ -769,6 +769,22 @@ func TestGameServerTcpUdpProtocol(t *testing.T) {
 	assert.Equal(t, "ACK TCP: Hello World !\n", replyTCP)
 }
 
+func TestGameServerWithoutPort(t *testing.T) {
+	t.Parallel()
+	gs := framework.DefaultGameServer(framework.Namespace)
+	gs.Spec.Ports = nil
+
+	_, valid := gs.Validate()
+	assert.True(t, valid)
+
+	readyGs, err := framework.CreateGameServerAndWaitUntilReadyWithOptionalPortCheck(framework.Namespace, gs, false)
+	if err != nil {
```

Comment:
Nit:

This `if err != nil` block can be changed to:
`require.NoError(err)`  (I know we have the `assert.FailNow()` pattern in a bunch of places, but that was before we had access to `require` :smile: )

Reply:
Done